test/utils/runners.go

```diff
@@ -132,6 +132,9 @@ type RCConfig struct {
 	PriorityClassName             string
 	TerminationGracePeriodSeconds *int64
 
+	// If set to true the framework will not check if some pod was deleted from deployment
+	AllowPodDeletions bool
```

Comment:
should this be grouped with MaxContainerFailures and follow a similar pattern (target number or percent instead of a boolean)?